config/manager/.key-server.conf

```diff
@@ -1,3 +0,0 @@
-# NOTE: update the Key server address beofere
-# deploying using `make deploy`
-KEY_SERVER=localhost:443
```

Comment:
Should `make deploy` take as an argument the key server address?

```make deploy KEY_SERVER=localhost:443```

Reply:
This file is supposed to consumed by the `kustomize` to generate a configmap with key server information. Yes we can make these as make variables and update this file inside the deploy target. This might also require additional fields but can be do able via make.